modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewSelectionModelImplTest.java

```diff
@@ -172,7 +172,7 @@ private TablePosition pos(int row, TableColumn<String,?> col) {
         assertFalse(model.isSelected(1, col0));
         assertFalse(model.isSelected(3, col0));
         assertFalse(cells(model), model.isSelected(3, null));
-        assertFalse(model.isSelected(3));
+        assertTrue(model.isSelected(3));
         assertEquals(1, model.getSelectedCells().size());
     }
 
```

Comment:
do we have tests isSelected(index) for all combinations of single/multiple/cellSelection? If not, now might be a good time to add them :)

Reply:
a *very* good point!
should definitely cover all the scenarios.